CHANGES.rst

```diff
@@ -12,6 +12,9 @@
   that argument. See `issue 208
   <https://github.com/zopefoundation/zope.interface/issues/208>`_.
 
+- When an invariant is defined in an interface, it's found by
+  `validateInvariants` in all interfaces inheriting from that interface.
+  Make sure to call each invariant only once when validating invariants.
```

Comment:
```suggestion
- Make sure to call each invariant only once when validating invariants. 
  Previously, invariants could be called multiple times because when an invariant is defined in an interface, it's found by in all interfaces inheriting from that interface.
   See `pull request 215 <https://github.com/zopefoundation/zope.interface/pull/215/>`_.
```

We try to write change log entries in imperative voice, i.e., starting with a verb.